scripts/sftp_upload.py

```diff
@@ -0,0 +1,23 @@
+# :coding: utf-8
+# :copyright: Copyright (c) 2014-2022 ftrack
+
+import ftrack_api.structure.id
+from sftp import SFTPAccessor
```

Comment:
@iwootten this should be updated to:

`from ftrack_sftp_accessor.sftp import SFTPAccessor`